linux_os/guide/services/fapolicyd/service_fapolicyd_enabled/rule.yml

```diff
@@ -23,3 +23,8 @@ references:
 ocil_clause: 'the service is not enabled'
 
 ocil: '{{{ ocil_service_enabled(service="fapolicyd") }}}'
+
+template:
+    name: service_enabled
+    vars:
+        servicename: fapolicyd
```

Comment:
Ansible remediation for service fapolicyd should not be generated. See `rhel8/templates/csv/services_enabled.csv`. That means you should add:
```
backends:
    ansible: off
```

Reply:
This should have worked, I'll look into what went wrong.

Reply:
Fixed, diff data can be seen in https://github.com/ComplianceAsCode/content/pull/4804/commits/90864b0279b38905391e29aebbc6fc0fbe29b96f